gecko/core/container.py

```
@@ -80,6 +100,12 @@ def __init__(self, parent: Optional["Container"] = None):
         self._parent = parent
         self._scoped_instances: Dict[Type, Any] = {}
         self._is_scope = parent is not None
+        # 用于检测解析时的循环依赖（保存正在解析的服务类型）
```

Ruff (0.14.7)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)